packages/react-components/react-tree/src/components/TreeItem/TreeItem.test.tsx

```diff
@@ -31,7 +28,7 @@ describe('TreeItem', () => {
   // TODO add more tests here, and create visual regression tests in /apps/vr-tests
 
   it('renders a default state', () => {
-    const result = render(<TreeItem>Default TreeItem</TreeItem>);
+    const result = render(<TreeItem itemType="branch">Default TreeItem</TreeItem>);
```

Comment:
```suggestion
    const result = render(<TreeItem itemType="leaf">Default TreeItem</TreeItem>);
```

If you want to keep existing snapshot as it is, probably it's better to use `leaf`